### ISSUE TITLE
Reuse trained weights on a new dataset with same number of classes

### ISSUE THREAD

**Issue by teodortotev** [AlexeyAB/darknet#5927]
After training Yolov4 on a personal dataset, I would like to reuse the trained weights and start a new training routine on a new dataset with the same number of classes (i.e. it is possible to reuse all of the weights). However, it seems that when provided with pre-trained weights (i.e. yolo-obj_best.weights) the network wants to continue training from the iteration it has stopped.

I tried modifying the config file, but then if I want a shorter training procedure the network thinks that it has already trained enough and simply saves the weights in /backup. 

I managed to start a training by setting a large number of max_batches and adjusting the learning rate scheduler to suit my need. However, this is not optimal and I would expect that there is a way to start a completely new training procedure with already trained weights. Please excuse me if this is available somewhere in documentation but I did not manage to find it.

**Comment by Luux** [comment]
Try the -clear flag when calling darknet :)

**Comment by teodortotev** [comment]
Yes, that did it. Thanks.
